packages/types/src/codec/create/createType.ts

```diff
@@ -17,13 +17,27 @@ function u8aHasValue (value: Uint8Array): boolean {
 // With isPedantic, actually check that the encoding matches that supplied. This
 // is much slower, but verifies that we have the correct types defined
 function checkInstance<T extends Codec = Codec, K extends string = string> (value: Uint8Array, created: FromReg<T, K>): void {
-  // For option, we are not adding  the initial byte (this is via storage)
-  const crHex = created.toHex(true);
+  // the underlying type created.toRawType()
+  const rawType = created.toRawType();
+
+  // ignore bytes completely - this is probably a FIXME, since these are somewhat
```

Comment:
This is probably "HACK"

 I wish I could spend time making this completely robust atm, but alas... it will need to wait